pkg/render/gatewayapi/gateway_api.go

```diff
@@ -347,10 +347,10 @@ func GatewayAPIResourcesGetter() func() *gatewayAPIResources {
 
 var GatewayAPIResources = GatewayAPIResourcesGetter()
 
-func GatewayAPICRDs(provider operatorv1.Provider) ([]client.Object, []client.Object) {
+func gatewayAPIK8sCRDs(provider operatorv1.Provider) (*gatewayAPIResources, []client.Object, []client.Object) {
```

Comment:
I don't think you need to change the function signature here - we never use the returned `resources` value.